containers/Investment/Tranche/index.tsx

```diff
@@ -50,7 +50,7 @@ class TrancheView extends React.Component<Props> {
         <Box>
           <Box margin={{ top: 'medium', bottom: 'large' }} >
             <Box>
-              <TrancheMetric tranche={tranche} investor={investor} type={tranche.type as any as TrancheType} />
+              <TrancheMetric tinlake={tinlake} tranche={tranche} investor={investor} type={tranche.type as any as TrancheType} />
```

Comment:
don't need to pass tinlake as a prop, if extending the tranche object with req token infos